dunwoody_disaster/views/MapScreen.py

```diff
@@ -123,9 +123,9 @@ def serialize(self) -> dict:
         return result
 
     @staticmethod
-    def from_json(json: dict, char: Character) -> "MapScreen":
+    def from_json(json: dict, char: Character) -> "Map":
```

Comment:
Just curious why some methods go from camelCase to snake_case

Reply:
Because I can't control my habits.

Reply:
If you want you can go through and create suggestions for each method and I'll click through and change them all.